### PR TITLE
[AIRFLOW-2063] Add missing & Fix existing docs for GCP

### PR DESCRIPTION
**Pull request by kaxil**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2063

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
- Add missing operator in `code.rst` and `integration.rst`
- Fix documentation in DataProc operator
- Fixed codeblocks and links in docstrings for GCP operators

One sample fix:
**Original**:
image

**Commit**:
image

Another Example in `DataStore Hook`:
**Original**:
image

**Fix**:
image

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
- Documentation update.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by kaxil** [comment]
cc @fenglu-g

File: airflow/contrib/hooks/datastore_hook.py [hunk 1]
```diff
@@ -66,7 +66,9 @@ def begin_transaction(self):
     def commit(self, body):
         """
         Commit a transaction, optionally creating, deleting or modifying some entities.
-        see https://cloud.google.com/datastore/docs/reference/rest/v1/projects/commit
+
+        .. seealso::
```
Comment (fenglu-g) [inline airflow/contrib/hooks/datastore_hook.py]: see also

File: airflow/contrib/hooks/datastore_hook.py [hunk 2]
```diff
@@ -77,7 +79,10 @@ def commit(self, body):
     def lookup(self, keys, read_consistency=None, transaction=None):
         """
         Lookup some entities by key
-        see https://cloud.google.com/datastore/docs/reference/rest/v1/projects/lookup
+
+        .. seealso::
```
Comment (fenglu-g) [inline airflow/contrib/hooks/datastore_hook.py]: see also

**Review by fenglu-db — APPROVED** [review]
LGTM, thanks @kaxil !

**Comment by Fokko** [comment]
Thanks for contributing to the docs, highly appreciated. LGTM, please squash the commits

**Comment by kaxil** [comment]
@Fokko You are welcome. I have also squashed the commits.

**Pull request closed without merging**